doc/userAnimation.dox

```diff
@@ -3,7 +3,7 @@
 
 Hopsan models can be animated, either in real-time or as replays. Animation is accessed through the Animate icon in the simulation toolbar, or from the menus.
 
-\htmlimagerightcaption{../graphics/animation_icon.png, Animate}
+\htmlimagerightcaption{animation_icon.png, Animate}
```

Comment:
Is it not possible to use relative path  graphics/animation_icon.png  (without the leading ../) then maybe we could just copy/move the graphics directory in the build step?

Reply:
I guess that would work too. But Doxygen already supports copying files using HTML_EXTRA_FILES command, so that's why I decided to use it.